pom.xml

```diff
@@ -82,7 +82,7 @@
     <hadoop.version>3.1.4</hadoop.version>
     <hbase.thirdparty.version>2.2.1</hbase.thirdparty.version>
 
-    <phoenix.thirdparty.version>1.0.0</phoenix.thirdparty.version>
+    <phoenix.thirdparty.version>1.1.0-SNAPSHOT</phoenix.thirdparty.version>
```

Comment:
we will replace the release version after the phoenix 3rd party right?
should we wait and then merge this?

Reply:
We shouldn't release 4.16/5.1with snapshot dependecies.

We can either merge this now, and update when we release thirdparty 1.1.0, or we can wait with the merge until we have 1.1.0 final.

I'm going to prepare an rc0 for thirdparty this weekend.

Reply:
@stoty if it follows the same step, I can release the thirdparty this in my time zone today.

Reply:
I will do publish rc0 and send out the mail  in an hour.

According to the ASF rules, we must leave at least 72 hours for a release vote.
If we take that literally, and count the weekend it, then we can do the official release monday afternoon PST.

Reply:
sounds great! Thanks again for the effort!